src/reputation/views/__init__.py

```diff
@@ -27,3 +53,77 @@ def distribute_rsc(request):
 
     response = Response({"data": f"Gave {amount} RSC to {user.email}"}, status=200)
     return response
+
+
+@api_view([GET])
+@permission_classes([AllowAny])
+def get_rsc_circulating_supply(request):
+    CACHE_KEY = f"{APP_ENV}_RSC_CIRCULATING_SUPPLY"
+
+    results = cache.get(CACHE_KEY)
+    if results:
+        return Response(results, status=200)
+
+    blockchain_supply = get_blockchain_rsc_supply()
+    inapp_supply = get_inapp_rsc_supply()
+    supply = {
+        "RSC Supply": f"{blockchain_supply:,f}",
```

Comment:
can you change this key to eth_blockchain

Reply:
Updated